x-pack/plugins/security_solution/public/timelines/components/timeline/body/index.test.tsx

```diff
@@ -23,7 +25,44 @@ import { timelineActions } from '../../../store/timeline';
 import { TimelineTabs } from '../../../../../common/types/timeline';
 import { defaultRowRenderers } from './renderers';
 
-jest.mock('../../../../common/lib/kibana');
+jest.mock('../../../../common/lib/kibana/hooks');
+jest.mock('../../../../common/hooks/use_app_toasts');
+
+jest.mock('../../../../common/lib/kibana', () => {
+  const originalModule = jest.requireActual('../../../../common/lib/kibana');
+  return {
+    ...originalModule,
+    useKibana: jest.fn().mockReturnValue({
+      services: {
+        application: {
+          navigateToApp: jest.fn(),
+          getUrlForApp: jest.fn(),
+        },
+        uiSettings: {
+          get: jest.fn(),
+        },
+        savedObjects: {
+          client: {},
+        },
+        timelines: {
```

Comment:
we can put this in a common timelines mock file so this and events_viewer can share it